src/alfasim_score/units.py

```diff
@@ -1 +1,7 @@
 LENGTH_UNIT = "m"
+DENSITY_UNIT = "kg/m3"
+SPECIFIT_HEAT_UNIT = "J/kg.K"
```

Comment:
```suggestion
SPECIFIC_HEAT_UNIT = "J/kg.K"
```

Remember to fix where it is used

Reply:
Do you mean Specific Heat Capacity?

Reply:
Fixed